packages/common/src/HttpClient.ts

```diff
@@ -26,6 +26,15 @@ export class HttpClient {
     return pingResponse
   }
 
+  // TODO: I am not sure returning an empty string is a good idea here. What data we may need from a 'relayBatchMode'?
+  async relayTransactionInBatch (relayUrl: string, request: RelayTransactionRequest): Promise<PrefixedHexString> {
+    const { error }: { signedTx: string, error: string } = await this.httpWrapper.sendPromise(relayUrl + '/relayBatchMode', request)
+    if (error != null) {
+      throw new Error(`Got error response from relay: ${error}`)
+    }
+    return ''
```

Comment:
I will look into it. It just seems weird that there is nothing a server would want to return here. Maybe once we implement the server we will have a better idea than "OK" string?

Reply:
since we have a requestId, it might be good to return it, just like normal sendTransaction returns (the client can calculate it locally, but its nice if the server comes up with the same result.